Update direct-routing-plan.md (#2123)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan.md

|||||

## Media traffic: Port ranges

> Note the requirements below apply if you ant to deploy Direct Routing without Media Bypass. For firewall requirements for Media Bypass please refer to [Plan for Media Bypass with Direct Routing](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-plan-media-bypass)

Reviewer: ```suggestion
> [!NOTE]
> These requirements apply if you want to deploy Direct Routing without Media Bypass. Learn more about Media firewall requirements for [Media Bypass with Direct Routing](https://docs.microsoft.com/microsoftteams/direct-routing-plan-media-bypass).
```
